identity-aware-proxy/IAPClient/README.md

```diff
@@ -0,0 +1,89 @@
+# Google Cloud Identity-Aware Proxy Samples
+
+This sample shows how to authenticate an http request when requesting a web page from
+a Google App Engine application protected by the [Google Cloud Identity-Aware Proxy](http://cloud.google.com/iap).
+
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+    https://www.visualstudio.com/), or the command line.
+
+## Build and Run
+
+1.  **Follow the instructions in the [root README](../../README.md)**.
+
+2.  Follow the instructions here
+    [https://cloud.google.com/iap/docs/app-engine-quickstart](https://cloud.google.com/iap/docs/app-engine-quickstart)
+    to create an App Engine application and enable Identity-Aware Proxy (IAP).
+
+3.  Follow the instructions here 
+    [https://cloud.google.com/iap/docs/authentication-howto](https://cloud.google.com/iap/docs/authentication-howto#iap-make-request-csharp)
+    to download service account credentials and observe your IAP client id.
+
+9.  From a Powershell command line, run to see command line arguments.
+    ```
+    PS > dotnet run
+    IAPClient 1.0.0
+    Copyright (C) 2018 IAPClient
+    ERROR(S):
+    Required option 'u, uri' is missing.
+
+    -j, --credentials    Path to your service account credentials .json file.
+
+    -u, --uri            Required. The URI to fetch.
+
+    -c, --iapcid         Your IAP client id listed on https://console.cloud.google.com/apis/credentials
+
+    --help               Display this help screen.
+
+    --version            Display version information.
+    ```
+    
+9.  And pass your own values:
+    ```
+    PS > dotnet run -- -u https://my-project.appspot.com `
+        -c 71377614389-vn0dmho3h2utdus0v3jq9dcf9foj1gph.apps.googleusercontent.com `

```

Comment:
can you obfuscate it or put "YOUR_OWN_CLIENTID" here?

Reply:
I don't see any security issues.  I like to put real-looking values in the code where possible.  That way, users can tell if their own client id is 42, then they probably don't have a real client id.